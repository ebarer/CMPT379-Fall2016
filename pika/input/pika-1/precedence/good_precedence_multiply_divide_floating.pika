exec {

  print 5.6 * 7.9 / 2.6, _n_.
  print 7.2 * 9.0 / 8.0, _n_.
  print 4.1 * 8.6 / 2.5, _n_.
  print 7.7 * 5.5 / 5.4, _n_.
  print 6.4 * 5.6 / 5.7, _n_.
  print 1.5 * 1.0 / 4.2, _n_.
  print 1.6 * 5.8 / 5.3, _n_.
  print 9.7 * 5.0 / 4.0, _n_.
  print 4.0 * 2.4 / 6.5, _n_.
  print 8.1 * 8.7 / 3.9, _n_.
  print 5.1 * 1.9 / 9.5, _n_.
  print 3.2 * 3.5 / 9.7, _n_.

  print 4.4 / 2.2 * 5.9, _n_.
  print 9.0 / 8.4 * 9.1, _n_.
  print 3.2 / 6.5 * 5.8, _n_.
  print 3.7 / 5.8 * 1.0, _n_.
  print 9.3 / 5.8 * 9.9, _n_.
  print 3.7 / 4.1 * 7.0, _n_.
  print 2.2 / 7.9 * 3.3, _n_.
  print 2.7 / 4.9 * 5.4, _n_.
  print 8.9 / 3.8 * 8.5, _n_.
  print 2.3 / 7.8 * 7.6, _n_.
  print 1.0 / 3.6 * 3.0, _n_.
  print 5.8 / 1.0 * 3.0, _n_.

  print 6.1 * 1.9 / 7.2 * 1.5, _n_.
  print 8.3 * 8.7 / 2.6 * 1.2, _n_.
  print 9.8 * 9.2 / 7.7 * 6.9, _n_.
  print 9.5 * 5.9 / 4.8 * 5.7, _n_.
  print 4.5 * 6.2 / 7.7 * 3.4, _n_.
  print 2.3 * 5.9 / 4.2 * 4.0, _n_.
  print 6.7 * 7.3 / 3.5 * 4.1, _n_.
  print 5.4 * 4.7 / 2.4 * 1.7, _n_.
  print 3.2 * 4.6 / 6.1 * 5.9, _n_.
  print 6.5 * 5.1 / 7.1 * 4.3, _n_.
  print 4.8 * 2.7 / 3.4 * 7.6, _n_.
  print 1.5 * 5.5 / 4.4 * 1.6, _n_.

  print 9.1 / 3.9 * 6.8 / 2.2, _n_.
  print 3.8 / 9.0 * 2.1 / 9.5, _n_.
  print 2.8 / 5.0 * 1.2 / 9.7, _n_.
  print 2.1 / 2.5 * 6.2 / 9.8, _n_.
  print 4.8 / 9.4 * 4.2 / 1.9, _n_.
  print 3.0 / 5.1 * 2.7 / 2.3, _n_.
  print 9.1 / 2.1 * 4.2 / 1.1, _n_.
  print 3.2 / 9.6 * 4.7 / 1.0, _n_.
  print 4.1 / 4.8 * 4.4 / 4.5, _n_.
  print 4.9 / 6.1 * 4.5 / 5.9, _n_.
  print 6.7 / 7.6 * 4.6 / 6.4, _n_.
  print 5.1 / 6.2 * 1.4 / 5.8, _n_.

}

