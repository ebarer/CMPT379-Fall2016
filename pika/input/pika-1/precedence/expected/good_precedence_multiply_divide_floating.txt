17.0154
8.1
14.104
7.84259
6.28772
0.357143
1.75094
12.125
1.47692
18.0692
1.02
1.15464
11.8
9.75
2.85538
0.637931
15.8741
6.31707
0.918987
2.97551
19.9079
2.24103
0.833333
17.4
2.41458
33.3277
80.7927
66.5594
12.3195
12.9238
57.2946
17.9775
14.2374
20.0768
28.9694
3
7.21212
0.0933333
0.0692784
0.531429
1.12878
0.690537
16.5455
1.56667
0.835185
0.61267
0.633635
0.198554
